src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -18,20 +19,34 @@
 @ExtendWith(MockitoExtension.class)
 @Tag(Unit.TAG)
 class SearchConceptsQueryBuilderTest {
+
+  private SnapshotBuilderDomainOption createTestSnapshotBuilderDomainOption(
+      String category, int id) {
+    SnapshotBuilderDomainOption domainOption = new SnapshotBuilderDomainOption();
+    domainOption.setCategory(category);
+    domainOption.setId(id);
+    return domainOption;
```

Comment:
Since `SnapshotBuilderDomainOption` supports fluent-style construction I don't think we need a method for this. In any event you can change this method to be:
```suggestion
    return new SnapshotBuilderDomainOption().category(category).id(id);
```